circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
```

Comment:
Oh this is just a first stab, we can always fix it later... I don't see a problem in merging in circle.yml, at least then we can start figuring out why it isn't running whenever we have the next PR.